lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx

```diff
@@ -111,13 +118,27 @@ class HelpVideoPanel extends React.Component {
               <Icon glyph={Icon.GLYPHS.play} />
             </button>
           </div>
-          <Spacing bottom={3} />
-          <Text bold heading>
+          <Spacing bottom={5} />
+          <Text
+            css={`
+              color: #404855;
+              font-size: 23px;
+              line-height: 31px;
+            `}
+            bold
+          >
             {this.props.title}
           </Text>
           <For each="desc" of={this.props.description}>
             <Spacing bottom={3} />
-            <Text medium>{desc}</Text>
+            <Text
+              css={`
+                color: #4d5766;
```

Comment:
Another hardcoded color

Reply:
Replaced this with `this.props.theme.textDarker` 😄